src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -219,6 +220,38 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     action_type = CLIArgumentType(
         help='The action of virtual network rule. Possible value is Allow.'
     )
+
+    enable_alw_type = CLIArgumentType(
+        arg_type=get_three_state_flag(), min_api='2021-06-01',
+        help='The account level immutability property. The property is immutable and can only be set to true '
+             'at the account creation time. When set to true, it enables object level immutability for all '
+             'the containers in the account by default.'
+    )
+
+    immutability_period_since_creation_in_days_type = CLIArgumentType(
+        options_list=['--immutability-period-in-days', '--immutability-period'], min_api='2021-06-01',
+        help='The immutability period for the blobs in the container since the policy creation, in days.',
+    )
+
+    account_immutability_policy_state_enum = self.get_sdk('models._storage_management_client_enums#AccountImmutabilityPolicyState',
+                               resource_type=ResourceType.MGMT_STORAGE)
+    immutability_policy_state_type = CLIArgumentType(
+        arg_type=get_enum_type(account_immutability_policy_state_enum),
+        options_list='--immutability-state', min_api='2021-06-01',
+        help='Defines the mode of the policy. Disabled state disables the policy, '
+        'Unlocked state allows increase and decrease of immutability retention time '
+        'and also allows toggling allow-protected-append-write property, '
+        'Locked state only allows the increase of the immutability retention time. '
+        'A policy can only be created in a Disabled or Unlocked state and can be toggled between the '
+        'two states. Only a policy in an Unlocked state can transition to a Locked state which cannot '
+        'be reverted.')
+
+    allow_protected_append_writes_type = CLIArgumentType(
+        arg_type=get_three_state_flag(), options_list=['--allow-append'], min_api='2021-06-01',
```

Comment:
```suggestion
        arg_type=get_three_state_flag(), options_list=['--allow-protected-append-writes', '--allow-append', '-w'], min_api='2021-06-01',
```
How about add another two options to keep consistent with container param definition?

Reply:
Sounds good. Should I add the full name for the other parameters?

Reply:
not necessary~